### PR TITLE
[T4A3][T15-B2] Ang Shi Ya

### PR DESCRIPTION
**Pull request by AngShiYa**

### CODE REVIEW

**Comment by AngShiYa** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.person;
+
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Contact.java:2]: Missing header comment. All non trivial methods should have javadoc format header comments.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    protected String value;
+    protected boolean isPrivate;
+    
+    public Contact() { }
+    
+    public Contact(String value, boolean isPrivate) {
+        this.value = value;
+        this.isPrivate = isPrivate;
+    }
+
```
Comment (K1ang) [inline src/seedu/addressbook/data/person/Contact.java:14]: This constructor can be used by the child classes to assign the variables instead.

**Comment by K1ang** [comment]
@AngShiYa 
Some comments added. Please close the PR after reading comments.
Good job keeping the tests updated upon modification of behavior.

**Pull request closed without merging**